File: HPWorkpath_v1.6.3/Samples/ExampleAPIServices/source/SuppliesSample/src/com/hp/workpath/sample/suppliessample/fragment/ResponseInterface.java
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.suppliessample.fragment;

import com.hp.workpath.api.Result;
import com.hp.workpath.api.supplies.supplyinfo.Supply;

import java.util.List;

public interface ResponseInterface {
    void success(List<Supply> supplies);
    void failure(String msg, Result result);
}

File: HPWorkpath_v1.6.3/Samples_Kotlin/ExampleAPIServices/source/SuppliesSample/src/com/hp/workpath/sample/suppliessample/fragment/ResponseInterface.kt
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.suppliessample.fragment

import com.hp.workpath.api.Result
import com.hp.workpath.api.supplies.supplyinfo.Supply

interface ResponseInterface {
    fun success(supplies: List<Supply>?)
    fun failed(msg: String?, result: Result?)
}